daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Relation.scala

```diff
@@ -18,9 +18,13 @@ object Relation {
   type Relation[A, B] = immutable.Map[A, Set[B]]
 
   object Relation {
+
     def merge[A, B](r: Relation[A, B], pair: (A, Set[B])): Relation[A, B] =
       r.updated(pair._1, r.getOrElse(pair._1, Set.empty[B]).union(pair._2))
 
+    def apply[A, B](pairs: Iterable[(A, Set[B])]): Relation[A, B] =
```

Comment:
Typically we'd use `ACollection.apply` with varargs. If you want to parse an `Iterable`, I think `Relation.from` might be a more Scala-esque name.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8139